.gitignore

```diff
@@ -23,6 +23,7 @@ var/
 *.egg-info/
 .installed.cfg
 *.egg
+.python-version
```

Comment:
Is there a reason this shouldn't be checked in? I think it's enough of a standard that it's useful to have, especially if we're all using pyenv